src/harness/testcases/testdata/esc_zone_contains_device_a.json

```diff
@@ -0,0 +1,18 @@
+{
+ "id" :"zone/exclusion_zone/ntia/2017_04_10/esc_zone_contains_deviceA_harness_test",
```

Comment:
I don't think the ID can have this format since the zone is not created by the NTIA. I'm not sure if there's any guidance on the ID format, so we can probably use a format like:

"zone/esc_zone/sas_admin/zone_id"